pkg/ebpf/verifier/stats.go

```diff
@@ -335,11 +330,15 @@ func generateLoadFunction(file string, opts *StatsOptions, results *StatsResult,
 						if progSourceMap == nil {
 							log.Printf("No source map found for program %s\n", progSpec.Name)
 						}
-						compl, err := unmarshalComplexity(p.VerifierLog, progSourceMap)
+						vlp := newVerifierLogParser(progSourceMap)
+						if vlp == nil {
```

Comment:
It doesn't look like this can fail, so this `if` check seems unnecessary?

Reply:
Right now yes, but I prefer to leave the check just in case the function changes and it becomes necessary to check.